### ISSUE TITLE
gpm failures should return non-zero value

### ISSUE THREAD

**Issue by drzraf** [getgrav/grav#3017]
When failing to install (as in the below example):
```
Preparing to install Form [v4.1.1]
  |- Downloading package...   100%
  |- Checking destination...  ok
  |- Installing package...  Installation aborted
```

the return value of the command should not be 0 so it can be caught.

Grav version 1.6.26

Context:
I'm provisioning instance using Ansible and (since https://github.com/getgrav/grav/issues/2397 is not yet there), run `./bin/gpm` commands. It currently can't catch gpm failures.

**Comment by mahagr** [comment]
I'm going to assign myself as I'm looking into similar stuff as this is.

**Comment by mahagr** [comment]
Should be fixed for the next version of Grav 1.7
